### PR TITLE
Send show_values key to static-viz js

### PR DESCRIPTION
**Pull request by adam-james-v**
The static viz components expect certain keys on the settings map passed into the js. At this time, the contract between the backend (Clojure) and the static-viz (js) feels a little under-specified. However, for the change to XY Charts here, we follow the existing method in code of checking the backend viz-settings map for some key, and passing it into js in a 'simplified' form. 

For instance, in this PR, `:graph.show-values` is taken and placed into the settings map as `:show_values` for the static-viz js to use.

### Before:
Values are not shown on the graph render, even though `:graph.show-values`
image

### After:
Values are shown on the graph render, but only when the setting is enabled.
image

### CODE REVIEW

**Comment by WiNloSt** [comment]
@adam-james-v This is looking really promising! But I find a single series line chart seems to still use the old implementation since it doesn't show any data point value. Would you mind taking a look at this?

## Email
localhost_ (2)

## Dashboard
localhost_3000_dashboard_2-my-dashboard (4)

File: src/metabase/pulse/render.clj
Comment (escherize) [inline src/metabase/pulse/render.clj:125]: I can't figure out why this condition needed to be moved down one. Could you explain it? I noticed it has a similar structure to this condition/result pair: https://github.com/metabase/metabase/pull/25116/files#diff-deeee17ae64e1f3ddb72660dbdd3179ce6a9a10f0bcce127c97af761320804e7R88-R95 I wonder if it would be a good idea to move it into there or not.

**Review by escherize — APPROVED** [review]

File: src/metabase/pulse/render.clj
Comment (adam-james-v) [reply src/metabase/pulse/render.clj:125]: I'm thinking about the best way to approach this. I actually want to put :line up in the first condition set, since we want to use combo-chart for the line impl too.

But, when I did that, tests were failing. I decided to keep things the same as they were before to try get things merged quickly to unblock some FE stuff.

Long story short, I'm looking at this now

Comment (adam-james-v) [reply src/metabase/pulse/render.clj:125]: In fact, I actually added this line back in in my second commit, and just added it in the wrong place. So this is entirely human error :/

**Pull request merged**
